Documentation/API.md

```diff
@@ -130,6 +130,10 @@ in your proto file:
 will generate a struct named `MyFooBar`.
 (Note: `swift_prefix` is only supported by protoc 3.2 or later.)
 
+:warning: Use of the `swift_prefix` is generally discourage as it means the
+message and enum names are even less like the ones generated for other
+languages (most leverage the proto package in some way).
```

Comment:
I don't really understand this last comment.  What "proto package" are you referring to?

Reply:
The `package` directive is the proto package - https://developers.google.com/protocol-buffers/docs/proto#packages